zeebe/exporters/camunda-exporter/src/test/java/io/camunda/exporter/CamundaExporterIT.java

```diff
@@ -382,4 +334,158 @@ private ExporterResourceProvider mockResourceProvider(
 
     return provider;
   }
+
+  @Nested
+  class ExportHandlerTests {
+
+    @TestTemplate
+    void shouldHandleExportedRecords(
+        final ExporterConfiguration config, final SearchClientAdapter clientAdapter)
+        throws IOException {
+      final var provider = new DefaultExporterResourceProvider();
+      provider.init(config);
+
+      final var exporter =
+          createExporter(
+              new HashSet<>(provider.getIndexDescriptors()),
+              new HashSet<>(provider.getIndexTemplateDescriptors()),
+              config);
+      for (final var handler : provider.getExportHandlers()) {
+        exportTest(exporter, handler, clientAdapter);
```

Comment:
💭 To make the test a bit understandable we might can concentrate on one handler at a time in the test. We could provide the different handlers via a parameter.